docs_source_files/content/remote_webdriver/remote_webdriver_client.en.md

```diff
@@ -81,7 +93,28 @@ caps.version = 67
 driver = Selenium::WebDriver.for :remote, :url => "http://www.example.com", :desired_capabilities => caps
   {{< / code-panel >}}
   {{< code-panel language="javascript" >}}
-// We don't have a JavaScript code sample yet -  Help us out and raise a PR  
+const { Builder, Capabilities } = require("selenium-webdriver");
+let chromeCapabilities = Capabilities.chrome();
+//Setting chrome options
+const capabilityName = 'goog:chromeOptions';
+var browserOptions = {
+	'args': [
+        "--start-maximized",
+        '--disable-gpu'
+	]
+};
+chromeCapabilities.set(capabilityName, browserOptions);
+(async function helloSelenium() {
+    let driver = new Builder()
+        .forBrowser("chrome")
+        .withCapabilities(chromeCapabilities)
+        .usingServer('http://localhost:4444/wd/hub').build();
```

Comment:
Hi @hiroksarker ,

Code looks awesome, but 'using server' may not be required unless we start the selenium-standalone-jar externally. We can target the code considering chrome/firefox drivers are set in path already. This is just a thought.  

Apologies if am wrong.

Reply:
Hi @Harsha509 
Thanks for the lovely thoughts. Just following previous language code example. I made the change again ‍as our target the code considering.